mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -83,3 +83,72 @@ Feature: Loans tests
     Then status 200
     And match response.itemStatusPriorToCheckIn == 'Checked out'
     And match response.itemId == itemId
+
+  Scenario: Get Loans collection
+    * def extInstanceTypeId = call uuid1
+    * def extInstitutionId = call uuid1
+    * def extCampusId = call uuid1
+    * def extLibraryId = call uuid1
+    * def extItemId1 = call uuid1
+    * def extItemId2 = call uuid1
+    * def limit = 5
+
+    #post items
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance') { extInstanceTypeId: #(extInstanceTypeId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostLocation') { extInstitutionId: #(extInstitutionId), extCampusId: #(extCampusId), extLibraryId: #(extLibraryId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: '989898', extItemId: #(extItemId1) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: '989899', extItemId: #(extItemId2) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostPolicies')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: '121212' }
+
+    # checkOut the first item
+    * def checkOutResponse1 = call read('classpath:domain/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: '121212', extCheckOutItemBarcode: '989898' }
+    # checkOut the second item
+    * def checkOutResponse2 = call read('classpath:domain/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: '121212', extCheckOutItemBarcode: '989899' }
+
+    # Get a paged collection of patron
+    Given path 'circulation/loans'
+    And param query = '(userId=="' + userId + '")'
+    And param limit = limit
```

Comment:
why do we need it ?

Reply:
When I reproduce on the UI firstly, I noticed the end point has some query parameters, so I also provided on this test as well

Reply:
please rid of from 'limit'. Honestly this endpoint has more query parameters(you can see all of them on https://s3.amazonaws.com/foliodocs/api/mod-circulation/circulation.html#circulation_loans_get) but all of them aren't obligatory and in our case we don't need them